src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/document/DataForTemplate.java

```diff
@@ -0,0 +1,4 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.document;
+
+public abstract class DataForTemplate {
```

Comment:
what does this class offer us?

Reply:
here:
https://github.com/hmcts/finrem-case-orchestration-service/pull/278/files#diff-369613aebaf9751a2c952a8171e983ecR8

it let us use a proper models instead of Maps